cmd/root.go

```diff
@@ -291,7 +291,6 @@ func (r *Root) execute() {
 	}
 
 	if err := rootCmd.Execute(); err != nil {
-		r.logger.Error("failed to start the CLI.", zap.Any("error", err.Error()))
```

Comment:
we need to check that only if the error is that a wrong command is used, only then we should avoid logging it, otherwise we need to log it.